test/ui-test/testSuites/suite_status/tst_groupChat/test.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Status Desktop Group Chat
+
+	As a user I want to use group chat functionality.
+
+  	Scenario: As an admin user I want to create a group chat with my contacts and invited users can send messages
+
+  	   Given the user starts the application with a specific data folder ../../../testSuites/suite_status/tst_groupChat/data
+  	    When the user tester123 logs in with password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user creates a group chat adding users
+        	 | Athletic |
+        	 | Nervous  |
+		Then the group chat is created
+		 And the group chat history contains created the group message
```

Comment:
```suggestion
		 And the group chat history contains the "group created" message
```

Reply:
I can use "" to have a better reading here. I agree it is not clear at all, but the message needs to be "created the group" bc the app says something like this: `admin user created the group Title of the chat`, so the test validates there is a message in the history that contains the given string.